tools/state-viewer/src/cli.rs

```diff
@@ -141,11 +141,20 @@ pub struct ChainCmd {
     start_index: BlockHeight,
     #[clap(long)]
     end_index: BlockHeight,
+    #[clap(long)]
+    show_full_hashes: bool,
```

Comment:
Please add a comment documenting this option.

Reply:
done.